### ISSUE TITLE
`get_code_for_mod_name` cannot load generic uploaders

### ISSUE THREAD

**Issue by erikyao** [biothings/biothings.api#289]
### Code

Branch: 0.12.x
Module: [Line #309, biothings/hub/dataplugin/assistant.py](https://github.com/biothings/biothings.api/blob/0.12.x/biothings/hub/dataplugin/assistant.py#L309)
Introduced since: commit 7c404eaa8031e4655002348b7daef7b71ec8bc9d

### Problem

Symptom: `self.get_code_for_mod_name(uploader_section["parser"])` cannot load generic uploaders, such as `"hub.dataload.data_parsers:load_obo"` in [`chebi` plugin](https://github.com/ericz1803/chebi).

Cause: On [Line #183](https://github.com/biothings/biothings.api/blob/0.12.x/biothings/hub/dataplugin/assistant.py#L183) of the definition of `def get_code_for_mod_name`, the `modpath` will become, say `"chebi.hub.dataload.data_parsers"`, an invalid module path for generic uploaders.

**Comment by newgene** [comment]
@erikyao looks like we can fix it this way (untested code):

```
        modpath = self.plugin_path_name + "." + mod
        try:
            pymod = importlib.import_module(modpath)
        except ImportError:
            pymod = importlib.import_module(mod) 
        # reload in case we need to refresh plugin's code
        importlib.reload(pymod)
```

Can you test, tweak as needed, in your case?

**Comment by erikyao** [comment]
@newgene fixed with commit 48f68372c26acf1a7c146ff88af13ea5615c93df
